.github/workflows/windows_unit_tests.yml

```diff
@@ -0,0 +1,47 @@
+name: Unit tests
+
+on:
+  push:
+    branches: ['**']
+  pull_request:
+    branches: ['**']
+
+envs:
+    PYTHONPATH: C:\nrn_test\lib\python
+    NEURONHOME: C:\nrn_test
+
+jobs:
+  build:
+
+    runs-on: ${{ matrix.os }}
+    strategy:
+      matrix:
+        os: [windows-latest]
+        python-version: [3.7, 3.8]
+
+    steps:
+      - uses: actions/checkout@v2
+      - uses: conda-incubator/setup-miniconda@v2
+        with:
+          activate-environment: test
+          python-version: ${{ matrix.python-version }}
+      - name: Install Neuron on Windows separately
+        shell: cmd
+        run: |
+          powershell -command "gh release download --repo https://github.com/neuronsimulator/nrn --pattern '*.exe' --output nrn-setup.exe"
+          start /b /wait .\nrn-setup.exe /S /D=C:\nrn_test
+          "C:\nrn_test\bin" >> $env:GITHUB_PATH
```

Comment:
During the install, neuron adds this to the PATH. But you have to start a new terminal to get the updated PATH variable. Since I couldn't figure out a way to restart the terminal on github actions, we could just append to the PATH manually

Reply:
Got it, that makes sense!